sources/common-dylan/tests/classes.dylan

```diff
@@ -6,9 +6,10 @@ Copyright:    Original Code is Copyright (c) 1995-2004 Functional Objects, Inc.
 License:      See License.txt in this distribution for details.
 Warranty:     Distributed WITHOUT WARRANTY OF ANY KIND
 
-/// Class test cases
-
-// Note: Locators are tested in system-test-suite.
+// TODO(cgay): Everything in this file tests things defined in dylan-extensions
+// and should be tested in a dylan-extensions-test-suite, not here, where they
```

Comment:
The fact that things are in dylan-extensions is normally an accident of the implementation, i.e. they are defined in the dylan library for sealing, run-time integration or other reasons, but with the intention of being re-exported from common-dylan or elsewhere. Making dylan-extensions too much of a "thing" unnecessarily exposes this sort of implementation detail.

Reply:
Fixed. PTAL (Please take another look)